test/protractor/spec.js

```diff
@@ -8,4 +8,52 @@ describe('PaperHive homepage', function() {
       'PaperHive · The coworking hub for researchers'
     );
   });
+
+    it('should be present subscribe form', function () {
+	var subscribeForm = element(by.name('subscribeForm'));
+	expect(subscribeForm.isPresent()).toBe(true);
+
+    });
+
+    it('should be present about us link', function () {
+	var aboutUs = element(by.linkText('About us'));
+	expect(aboutUs.isPresent()).toBe(true);
+
+	var linkAboutUs = element(by.xpath('/html/body/footer/ng-include/div[1]/div/div/div[2]/ul/li[1]/a'));
+	linkAboutUs.click();
+	expect(browser.getCurrentUrl()).toBe('http://localhost:8080/about');
```

Comment:
mmm I can imagine that if in the protractor.js we use the base URL of the jenkins, then it will work.
Since I'm running the tests locally I suppose that I will need to have in localhost, but once I've tested it I can push it with the correct baseURL.
Does it make sense?